include/verilated_types.h

```diff
@@ -70,6 +71,67 @@ extern std::string VL_TO_STRING_W(int words, const WDataInP obj);
 #define VL_OUT(name, msb, lsb) IData name  ///< Declare output signal, 17-32 bits
 #define VL_OUTW(name, msb, lsb, words) VlWide<words> name  ///< Declare output signal, 65+ bits
 
+//===================================================================
+// Activity trigger vector
+
+template <std::size_t T_size>  //
+class VlTriggerVec final {
+private:
+    // MEMBERS
+    std::array<bool, T_size> m_flags;  // State of the assoc array
+    static_assert(sizeof(m_flags) >= sizeof(bool) * T_size, "We don't want a special array");
```

Comment:
Not sure what you are checking for - reword assertion text please

Reply:
We want each `bool` to be directly byte addressable for performance, so we don't want this to be something "clever" like a bit-packed thing, though TBH, it probably does not matter (or migt even be beneficial, as there are a lot more reads than writes). Will revisit and fix.

Reply:
Removed as this is just premature optimization.